velox/exec/Task.cpp

```diff
@@ -1532,6 +1532,7 @@ std::string Task::toString() const {
   }
 
   out << " drivers:\n";
+  std::lock_guard<std::mutex> l(mutex_);
```

Comment:
Shall we move this lock acquistion to l1526?

Reply:
Makes sense. I've moved it up. Thank you for the suggestion!